README.md

```diff
@@ -1,7 +1,41 @@
 ## Kittl/Vectorizing
 
+[![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/Kittl/vectorizing)
+
 Utility to vectorize raster images :rocket:
 
+## Local development
+
+1. Install python development tools:
+	- Open this repository in the dev container:
+		- Install [`dev container`](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension in vscode
+		- At the top of the repo click the `Open in dev container` badge, or:
+			- In vscode press `cmd+shift+p` and search for the command: `Open folder in container`
+			- The dev container has already installed all necessary tools: `conda`, `black`, `flake8`, `pre-commit`, `AWS CLI`, `Act`
+	- Install manually:
+		- [Conda](https://docs.conda.io/projects/miniconda/en/latest/miniconda-install.html)
+
+2. Install system dependencies for `pypotrace`:
+
+	```bash
+	sudo apt-get update
+	sudo apt-get install build-essential python-dev libagg-dev libpotrace-dev pkg-config
+	```
+
+3. Create the conda development environment and activate it
+
+	```bash
+	conda env create -n dev -f envs/dev.yaml
```

Comment:
I am stuck in this step, when I run `conda env create -n envs/dev.yaml` with the error:

```
EnvironmentFileNotFound: '/Users/cristian/vectorizing/environment.yml' file not found
```

Is there a previous step missing, where we create `environment.yml`?